.gitignore

```diff
@@ -301,3 +301,10 @@ TSWLatexianTemp*
 #*Notes.bib
 
 old
+
+# 忽略所有的 .DS_Store 文件
+.DS_Store
+
+# 忽略 papers 目录下的 _book 目录，不忽略主 cadp_docs 目录下的 _book
+/papers/_book/
```

Comment:
Will this work for all subdirectories? E.g., `/papers/paper1/_book`, `/papers/paper2/_book`.

Also, I think you may have missed the full intention of this. The point is to not include any rendered/compiled files in the source. So that also includes the generated `lib` directory and other artifacts, like `_bookdown_files`. Let me know if that is unclear.

Reply:
Sorry Professor Rowe, I'm still figuring out how to create and use .gitignore and accidentally clicked on the pull request. Thanks for your reply, I'll try it out and email you if I have any questions.

Reply:
No problem @Zhengnan817